src/analysis/normed_space/ordered.lean

```diff
@@ -1,15 +1,34 @@
 /-
 Copyright (c) 2020 Anatole Dedecker. All rights reserved.
+Copyright (c) 2021 Christopher Hoskin. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Anatole Dedecker
+Authors: Anatole Dedecker, Christopher Hoskin
 -/
 import analysis.normed_space.basic
+import algebra.lattice_ordered_group
+import topology.order.lattice
```

Comment:
Could you move all this new content to its own fiile? I'd suggest `analysis.normed_space.lattice`.

Reply:
Done. I've gone with `analysis.normed_space.lattice_ordered_group`.